docs/data_driven_testing.adoc

```diff
@@ -371,6 +371,19 @@ Data tables, data pipes, and variable assignments can be combined as needed:
 include::{sourcedir}/datadriven/DataSpec.groovy[tag=combined-variable-assignment]
 ----
 
+== Type Coercion for Data Variable Values
+
+Data variable values are casted to the declared parameter type using
```

Comment:
```suggestion
Data variable values are coerced to the declared parameter type using
```
otherwise the verb `cast` is irregular https://www.usingenglish.com/reference/irregular-verbs/cast.html but I'm not 100% sure if that applies to the programming term.